docs/upgrade/services_removal.md

```diff
@@ -0,0 +1,92 @@
+# Services removal
+
+We removed services (i.e. *Service classes - not to be interchanged with Symfony services) from our model and moved logic into more suitable places.
```

Comment:
There should be info about when this occured e.g. link to changelog or to upgradelog or hwy this page exists. Without this it looks like important part of docs. But I feel it is just more info for upgrading...

Reply:
I added a link to PR, is it better?